subsys/net/lib/coap_utils/coap_utils.c

```diff
@@ -59,7 +61,7 @@ static void coap_receive(void)
 	int len;
 	int ret;
 
-	while (1) {
+	while (is_active) {
```

Comment:
if is_active is false no value is returned

Reply:
`is_active` is set on `true` before `coap_receive` thread is created and starts thus there is no bug here

Reply:
... Sorry my bad. The github code folding hid too much code:)